libs/blocks/chart/chart.js

```diff
@@ -116,28 +150,43 @@ export const getChartOptions = (chartType, dataset, colors, unit = '') => {
 };
 
 const init = async (el) => {
+  const parent = el.parentElement;
   const children = el?.querySelectorAll(':scope > div');
   children[0]?.classList.add('chart_title');
   children[1]?.classList.add('chart_subTitle');
   children[3]?.classList.add('chart_footnote');
 
-  // ToDo: Replace hardcoded size MWPW-112994
-  const size = LARGE;
+  // Add chart container
+  const container = document.createElement('div');
+  container.className = 'chart_container';
+  container.append(...children);
+  el.appendChild(container);
+
+  // Grab section metadata
+  const chartStyles = parent.className.split(' ').slice(1);
+  parent.classList.remove(...chartStyles);
+
+  const size = chartStyles?.find(style => style === SMALL || style === MEDIUM || style === LARGE);
+  el.classList.add(`chart_${size}`);
+
+  const chartType = chartTypes?.find((type) => el?.className?.indexOf(type));
+  el.classList.remove(chartType);
```

Comment:
I'm unclear on why you need to remove the classes

Reply:
I guess i'm unclear as to why we would intend to keep it in? The class `bar` that gets added is not defined. Similarly, I remove the classes that get added to the top-level div from the section metadata block. For example, I just need to parse the color value and does not need to be in the css. Please let me know if I'm confusing anything.

Reply:
Every bit of code we add increases the load time, so if the classes aren't doing any harm being there, then we can just leave them

Reply:
Ok thanks, good to know. I'll go ahead and get rid of the code logic to remove classes if they're not doing any harm being there.